chord_metadata_service/mohpackets/docs/schema_manual.yml

```
@@ -420,16 +460,17 @@ components:
       - program_id
       - submitter_donor_id
       - submitter_treatment_id
-      title: ChemotherapyIngestSchema
+      - systemic_therapy_type
+      title: SystemicTherapyIngestSchema
       type: object
-    ChemotherapyModelSchema:
+    SystemicTherapyModelSchema:
       properties:
         actual_cumulative_drug_dose:
           anyOf:
           - type: integer
```

This might have been an issue not picked up before but the spreadsheet specifies this as 'number' rather than 'integer' so not sure we can assume non decimal numbers
```suggestion
          - type: number
```